src/core/qgssettings.cpp

```diff
@@ -177,12 +177,13 @@ QString QgsSettings::fileName() const
 
 void QgsSettings::sync()
 {
-  return mUserSettings->sync( );
+  return mUserSettings->sync();
 }
 
 void QgsSettings::remove( const QString &key )
 {
-  mGlobalSettings->remove( key );
```

Comment:
Fixed in https://github.com/qgis/QGIS/commit/e045b369cf6cb5700be8cc783f32f8af06c93485